src/services/dumper/dumper.js

```diff
@@ -135,93 +91,82 @@ class Dumper {
       dependencies,
     };
 
-    this.writeFile(projectPath, 'package.json', `${JSON.stringify(pkg, null, 2)}\n`);
+    this.writeFile('package.json', `${JSON.stringify(pkg, null, 2)}\n`);
   }
 
   static tableToFilename(table) {
     return _.kebabCase(table);
   }
 
-  static getDatabaseUrl(config) {
+  static getDatabaseUrl(dbConfig) {
```

Comment:
This could be in a utils function, and moved to the parent level as both of the dumpers will require it.

Reply:
Done